solr/solr-ref-guide/src/updating-parts-of-documents.adoc

```diff
@@ -103,6 +103,108 @@ The resulting document in our collection will be:
 }
 ----
 
+=== Updating Child Documents
+
+Solr supports modifying, adding and removing child documents as part of atomic updates. +
+Schema and configuration requirements are detailed in
+<<indexing-nested-documents#schema-configuration, Indexing Nested Documents>>. +
+Under the hood, Solr retrieves the whole nested structure, deletes the old documents,
+and re-indexes the structure after applying the atomic update. +
+Syntactically, nested/partial updates are very similar to a regular atomic update,
+as demonstrated by the examples below.
+
+[NOTE]
+====
+.\_route_ Param
+To ensure each nested update is routed to its respective shard,
+`\_route_` param should be set to the root document's ID.
+====
+
+If the following document exists in our collection:
+
+[source,json]
+----
+{
+ "id":"mydoc",
+ "product":"T-Shirt",
+ "stock": {
+           "id":"mydoc2",
+           "color":"red",
+           "size": ["L"]
+           }
+}
+----
+
+And we apply the following update command, setting `\_route_`=mydoc:
+
+[source,json]
+----
+{
+ "id":"mydoc",
+ "stock": {
+           "add":
+                 {
+                  "id":"mydoc3",
+                  "color":"blue",
+                  "size": ["M"]
+                 }
+          }
+}
+----
+
+The resulting document in our collection will be:
+
+[source,json]
+----
+{
+ "id":"mydoc",
+ "product":"T-Shirt",
+ "stock": [{
+            "id":"mydoc2",
+            "color":"red",
+            "size": ["L"]
+           },
+           {
+            "id":"mydoc3",
+            "color":"blue",
+            "size": ["M"]
+           }]
+}
+----
+
+Documents inside nested structures can also be updated, provided `\_route_` is set to the root document's ID
```

Comment:
RE "provided"... that wording suggests \_route_ is optional.  It's NOT, right?  Then instead say nothing or just remind the user what it should be.